helm-chart/kuberay-operator/values.yaml

```diff
@@ -50,7 +45,6 @@ readinessProbe:
   periodSeconds: 5
   failureThreshold: 5
 
-createCustomResource: true
```

Comment:
Do you think we should consider keeping this and actually implementing it?
(If the flag is False, don't create CRDs.)
There could be some good reasons to install the operator without creating the CRDs
(for example, there could be different levels of admin permissions required to install the CRDs, so installing the operator and installing the CRDs could be handled as different steps)

Reply:
Good idea, I will update the PR.

Reply:
I found that Helm has a built-in `--skip-crds` flag. 

From https://helm.sh/docs/chart_best_practices/custom_resource_definitions/
"""
With the arrival of Helm 3, we removed the old crd-install hooks for a more simple methodology. There is now a special directory called crds that you can create in your chart to hold your CRDs. These CRDs are not templated, but will be installed by default when running a helm install for the chart. If the CRD already exists, it will be skipped with a warning. If you wish to skip the CRD installation step, you can pass the `--skip-crds` flag.
"""

Updated the document.